src/coreclr/jit/importer.cpp

```diff
@@ -13732,11 +13783,29 @@ void Compiler::impImportBlockCode(BasicBlock* block)
                 // See also identical code in impGetByRefResultType and STSFLD import.
                 if (varTypeIsI(op1) && (genActualType(op2) == TYP_INT))
                 {
-                    op2 = gtNewCastNode(TYP_I_IMPL, op2, uns, TYP_I_IMPL);
+#ifdef TARGET_LOONGARCH64
```

Comment:
Please remove this. There should be no architecture-specific code in the importer.

Reply:
had removed.
Thanks